.github/stale.yml

```diff
@@ -0,0 +1,33 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 7
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - "Feature Request"
+  - "WIP"
+  - "Bug"
+  - pinned
```

Comment:
Why is pinned not labeled?

Reply:
@yousefhamza I _think_ pinned here refers to this https://docs.github.com/en/issues/tracking-your-work-with-issues/pinning-an-issue-to-your-repository

Reply:
We have other "labels" already to control the bot behavior, adding `pinned` here adds _infelexibility_; in case of a pinned issue that we want to be opened/closed, we can add/remove the labels we already defined instead of being locked into opened.

Reply:
@yousefhamza ok got you